src/SFA-DAS-HMRC-API-Stub/Configuration/FractionCalDateConfigExtension.cs

```diff
@@ -0,0 +1,30 @@
+﻿using Microsoft.Azure.Documents.Client;

```

Comment:
I would use the same config extension for both the Calc and the Fractions themselves as they logically sit together